helm/alfresco-content-services/templates/deployment-repository.yaml

```diff
@@ -84,6 +84,15 @@ spec:
           - name: data
             mountPath: {{ .Values.repository.persistence.data.mountPath }}
             subPath: {{ .Values.repository.persistence.data.subPath }}
+          {{- if .Values.repository.licenseSecret }}
+          - mountPath: /usr/local/tomcat/shared/classes/alfresco/extension/license/
+            name: acs-license
```

Comment:
I remember to have struggle on license recently but I was thinking I was doing something wrong... 🤦🏻